pkg/controllers/allocation/bind.go

```diff
@@ -61,11 +61,11 @@ func (b *Binder) Bind(ctx context.Context, node *v1.Node, pods []*v1.Pod) error
 	}
 
 	// 4. Bind pods
-	logging.FromContext(ctx).Infof("Binding %d pod(s) to node %s", len(pods), node.Name)
 	errs := make([]error, len(pods))
 	workqueue.ParallelizeUntil(ctx, len(pods), len(pods), func(index int) {
 		errs[index] = b.bind(ctx, node, pods[index])
 	})
+	logging.FromContext(ctx).Infof("Bound %d pod(s) to node %s", len(pods), node.Name)
```

Comment:
Let's say that we had to bind 10 pods, and only 5 succeeded. This would log that we binded 10 pods, right? Can we change this to be more accurate?